### PR TITLE
turkish Language Support 

### PR DESCRIPTION
**Pull request by neoplus72**

### CODE REVIEW

File: app/src/main/res/values-tr/strings.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<resources>
+    <string name="app_name">Wattz</string>
+    <string name="back">Geri</string>
+    <string name="battery">Pil</string>
+    <string name="charging">Doluyor</string>
+    <string name="chargingSince">Şu tarihten beri şarj ediliyor:/string>
```
Comment (dubrowgn) [inline app/src/main/res/values-tr/strings.xml:6]: `:/string>` -> `</string>`?

File: app/src/main/res/values-tr/strings.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    <string name="power">Güç</string>
+    <string name="settings">Ayarlar</string>
+    <string name="temperature">Sıcaklık</string>
+    <string name="timeToFullCharge">Tam Şarj Süresi</string>
+    <string name="voltage">Gerilim</string>
+    <string name="yes">Evet</string>
+
+    <!-- settings -->
+    <string name="currentScalar">Güç Skalası Geçici Çözümü</string>
+    <string name="currentScalarDesc">Fişe takılı değilken, bu skalayı şuna kadar değiştirin: \u0022Power\u0022 is greater than 0W, but less than about 10W.</string>
```
Comment (dubrowgn) [inline app/src/main/res/values-tr/strings.xml:20]: This looks half done. Can you translate the second half?

File: app/src/main/res/values-tr/strings.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    <string name="invertCurrent">Şarj Geçici Çözümü</string>
+    <string name="invertCurrentDesc">Fişe takılı değilken, aşağıdaki durumlarda bu geçişi etkinleştirin: \u0022Charging\u0022 says \u0022yes\u0022.</string>
```
Comment (dubrowgn) [inline app/src/main/res/values-tr/strings.xml:22]: `"Charging" says "yes"` needs to be translated.

File: app/src/main/res/values-tr/strings.xml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    <string name="workarounds">Geçici çözümler</string>
+    <string name="workaroundsDesc">Çoğu telefon, Android BatteryManager arayüzünü doğru bir şekilde uygulamaz ve varsayılan olarak yanlış pil ölçümlerini bildirir.  Gerekirse, telefonunuzdaki uygulama kusurlarını gidermek için aşağıdaki ayarları kullanın.</string>
```
Comment (dubrowgn) [inline app/src/main/res/values-tr/strings.xml:24]: I don't speak Turkish, so forgive me if this doesn't make sense. Plugging this into Google/Bing translate to produce English results in "use the settings below to work around app errors on your phone." It's important to be clear that these issues are caused by the phone manufacturer, not the app. Is there an alternative translation that makes this more clear?

I also appreciate that Google/Bing translate could easily be wrong, so let me know if that's the case here.

**Review by dubrowgn — CHANGES_REQUESTED** [review]
This is great, thanks for raising it!

There are a few problems I spotted, and the screenshots all seem to be empty.

Out of curiosity, how were these generated?

File: app/src/main/res/values-tr/strings.xml (hunk 4 above)
Comment (neoplus72) [reply app/src/main/res/values-tr/strings.xml:24]: When the changes you want are translated into my language, their meaning will be wrong. 

Bing translate, google translate, & programs will give wrong results

File: app/src/main/res/values-tr/strings.xml (hunk 2 above)
Comment (neoplus72) [reply app/src/main/res/values-tr/strings.xml:20]: Okay

File: app/src/main/res/values-tr/strings.xml (hunk 3 above)
Comment (neoplus72) [reply app/src/main/res/values-tr/strings.xml:22]: Okay

**Pull request closed without merging**